### PR TITLE
fix: adjust payee predictions after mn_rr activation, add tests

### PR DESCRIPTION
**Pull request by UdjinM6**
## Issue being fixed or feature implemented
Payment predictions in GUI are wrong when mn_rr is active, `masternode winners` RPC is affected by the same issue too. Actual payments aren't affected.

## What was done?
Adjust calculations, add tests for `masternode winners`.

## How Has This Been Tested?
Run dash-qt on testnet, check "Next Payment" on "Masternode" tab. Run tests.

## Breaking Changes
n/a

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: test/functional/feature_llmq_evo.py [hunk 1]
```diff
@@ -248,6 +250,40 @@ def test_masternode_count(self, expected_mns_count, expected_evo_count):
         assert_equal(detailed_count['regular']['total'], expected_mns_count)
         assert_equal(detailed_count['evo']['total'], expected_evo_count)
 
+    def test_masternode_winners(self, v20active=False):
```
Comment (knst) [inline test/functional/feature_llmq_evo.py:253]:
```suggestion
    def test_masternode_winners(self, mnrr_active=False):
```

**Review by knst — APPROVED** [review]
utACK 3145d144358d47da14a9764d4b745697b0dd72a1

**Review by PastaPastaPasta — COMMENTED** [review]
please squash 3145d144358d47da14a9764d4b745697b0dd72a1 into original commit(s) otherwise looks good

**Review by ogabrielides — APPROVED** [review]

**Review by PastaPastaPasta — APPROVED** [review]
utACK 715bc1a

**Pull request merged**